Samples/PowerShell/NTA.AddCBQoSSources.ps1

```diff
@@ -0,0 +1,57 @@
+# This sample scripts demonstrates how to add enabled CBQoS sources 
+# with required Orion node and interfaces. Interfaces are discovered with NPM 
+# discovery feature.
+# You need to be logged as a user with enabled Allow Node Management Rights.
+
+
+# Connect to SWIS
+$hostname = "swis-machine-hostname"                     # Update to match your configuration
+$username = "admin"                                     # Update to match your configuration
+$password = New-Object System.Security.SecureString     # Update to match your configuration
+$cred = New-Object -typename System.Management.Automation.PSCredential -argumentlist $username, $password
+$swis = Connect-Swis -host $hostname -cred $cred
```

Comment:
I feel like it's more straightforward to use an empty `-Password` if that's what you're shooting for here. I don't think it's obvious that a null `System.Security.SecureString` object is going to achieve that effect.

Reply:
Empty `System.Security.SecureString`  and credential object for swis connection is used in almost all other examples. I was thinking that this is a suggested way how to create connection and I wanted to keep my examples in line with others.

Reply:
That makes a lot of sense. All the examples need to be freshened IMHO, but that's a PR for another day. I agree with staying consistent for now.